Japx.podspec

```diff
@@ -1,96 +1,56 @@
+  s.source = { :git => "https://github.com/infinum/Japx.git", :tag => s.version.to_s }
 
   s.requires_arc = true
   s.platform = :ios, :osx
-  s.ios.deployment_target = '10.0'
-  s.osx.deployment_target = '10.12'
-  s.swift_version = '5.0'
+  s.ios.deployment_target = "10.0"
+  s.osx.deployment_target = "10.12"
+  s.swift_version = "5.0"
 
-  s.default_subspec = 'Core'
-
-  s.subspec 'Core' do |sp| 
-    sp.source_files = 'Japx/Classes/Core/**/*'
-    sp.frameworks = 'Foundation'
-  end
+  s.default_subspec = "Core"
 
-  s.subspec 'Codable' do |sp| 
-    sp.source_files = 'Japx/Classes/Codable/**/*'
-    sp.dependency 'Japx/Core'
+  s.subspec "Core" do |sp|
+    sp.source_files = "Japx/Classes/Core/**/*"
+    sp.frameworks = "Foundation"
   end
 
-  s.subspec 'Alamofire' do |sp| 
-    sp.source_files = 'Japx/Classes/Alamofire/**/*'
-    sp.dependency 'Japx/Core'
-    sp.dependency 'Alamofire'
+  s.subspec "Alamofire" do |sp|
+    sp.source_files = "Japx/Classes/Alamofire/**/*"
+    sp.dependency "Japx/Core"
+    sp.dependency "Alamofire"
   end
 
-  s.subspec 'RxAlamofire' do |sp| 
-    sp.source_files = 'Japx/Classes/RxAlamofire/**/*'
-    sp.dependency 'Japx/Alamofire'
-    sp.dependency 'RxSwift'
+  s.subspec "RxAlamofire" do |sp|
+    sp.source_files = "Japx/Classes/RxAlamofire/**/*"
+    sp.dependency "Japx/Alamofire"
+    sp.dependency "RxSwift"
   end
 
-  s.subspec 'CodableAlamofire' do |sp| 
-    sp.source_files = 'Japx/Classes/CodableAlamofire/**/*'
-    sp.dependency 'Japx/Alamofire'
-    sp.dependency 'Japx/Codable'
+  s.subspec "Moya" do |sp|
+    sp.source_files = "Japx/Classes/Moya/**/*"
+    sp.dependency "Japx/Core"
+    sp.dependency "Moya/Core"
   end
 
-  s.subspec 'RxCodableAlamofire' do |sp| 
-    sp.source_files = 'Japx/Classes/RxCodableAlamofire/**/*'
-    sp.dependency 'Japx/CodableAlamofire'
-    sp.dependency 'Japx/RxAlamofire'
-  end
-  
-  s.subspec 'Moya' do |sp|
-    sp.source_files = 'Japx/Classes/Moya/**/*'
-    sp.dependency 'Japx/Core'
-    sp.dependency 'Moya/Core'
-  end
-  
-  s.subspec 'RxMoya' do |sp|
-    sp.source_files = 'Japx/Classes/RxMoya/**/*'
-    sp.dependency 'Japx/Moya'
-    sp.dependency 'Moya/RxSwift'
-  end
-  
-  s.subspec 'CodableMoya' do |sp|
-    sp.source_files = 'Japx/Classes/CodableMoya/**/*'
-    sp.dependency 'Japx/Core'
-    sp.dependency 'Japx/Codable'
-    sp.dependency 'Japx/Moya'
-  end
-  
-  s.subspec 'RxCodableMoya' do |sp|
-      sp.source_files = 'Japx/Classes/RxCodableMoya/**/*'
-      sp.dependency 'Japx/Core'
-      sp.dependency 'Japx/CodableMoya'
-      sp.dependency 'Japx/RxMoya'
+  s.subspec "RxMoya" do |sp|
+    sp.source_files = "Japx/Classes/RxMoya/**/*"
+    sp.dependency "Japx/Moya"
+    sp.dependency "Moya/RxSwift"
   end
 
-  s.subspec 'ObjC' do |sp|
-      sp.source_files = 'Japx/Classes/ObjC/**/*'
-      sp.dependency 'Japx/Core'
+  s.subspec "ObjC" do |sp|
```

Comment:
Why did you change everything to double quotes?
I was under the impression single quotes should be used in ruby as a default, and double only if we need string interpolation 🤔

Reply:
Not sure to be honest, I didn't notice it - probably my ruby linter in VS code changed it to double quotes